### PR TITLE
Added Mission

### PR DESCRIPTION
**Pull request by tmbutterworth**
Added mission for assisting disabled navy ships.

### CODE REVIEW

File: data/boarding missions.txt [hunk 1]
```diff
@@ -37,3 +37,34 @@ mission "Assisting Hai"
...
+	destination Farpoint
+
+
+mission "Assisting Militia"
+	assisting
+	repeat
+	to offer
+		random < 27
+	source
+		government Escort
```
Comment (jafdy) [inline data/boarding missions.txt]: So you get paid for repeatedly disabling your own ships and then fixing them? Do you perhaps mean `government "Free Worlds"`?

Comment (tehhowch) [reply data/boarding missions.txt]: Or `government Militia`

**Comment by tmbutterworth** [comment]
> Did you review the previous pull requests that proposed assisting missions for various groups? I would have expected some sort of discussion in your OP if you had.
> 
> See below and also #3707 and #333

I didn't realize there was other ones I'll take a look.

**Comment by tmbutterworth** [comment]
> Did you review the previous pull requests that proposed assisting missions for various groups? I would have expected some sort of discussion in your OP if you had.
> 
> See below and also #3707 and #333

I reviewed them. They are not really alike. One was trying to make a Join navy mission. One was attempting to transport injured to a hospital.

Mine are just a simple payments with lore friendliness. No added mission or ridiculous high prize.

File: data/boarding missions.txt [hunk 2]
```diff
@@ -37,3 +37,50 @@ mission "Assisting Hai"
...
+
+
+mission "Assisting Free Worlds"
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government "Free Worlds"
+		has "event joined the free worlds"
```
Comment (Amazinite) [inline data/boarding missions.txt]: 1) Why?
2) `"event: joined the free worlds"`

Comment (tehhowch) [reply data/boarding missions.txt]: In addition to the erroneous condition text, conditions are not valid criteria for `LocationFilter`. They belong in a `ConditionSet`.

**Comment by Amazinite** [comment]
>One was trying to make a Join navy mission. One was attempting to transport injured to a hospital.

And what of #2999?

**Comment by tmbutterworth** [comment]
> > One was trying to make a Join navy mission. One was attempting to transport injured to a hospital.
> 
> And what of #2999?

That PR had a large number of boarding missions, but it is closed and was never merged. If it was still a candidate for merging mine would not be necessary. That PR had a lot of conflict due to wording, lore, etc. Mine are simple and not complex.

**Comment by toilethinges** [comment]
Greatful is not a word.

Greatful -> Grateful

**Comment by tmbutterworth** [comment]
> Greatful is not a word.
> 
> Greatful -> Grateful

Thanks, fixed it.

File: data/boarding missions.txt [hunk 3]
```diff
@@ -37,3 +37,48 @@ mission "Assisting Hai"
...
+	assisting
+	repeat
+	to offer
+		random < 27
+	source
+		government Republic
+	on offer
+		payment 25000
+		conversation
+			`The captain of the <origin> is grateful for your service to The Navy and gives you <payment>.`
```
Comment (Amazinite) [inline data/boarding missions.txt]: `The` before `Navy` shouldn't be capitalize.

File: data/boarding missions.txt [hunk 4]
```diff
@@ -37,3 +37,48 @@ mission "Assisting Hai"
...
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government "Free Worlds"
+	on offer
+		payment 10000
+		conversation
+			`The captain of the <origin> is grateful for your service to The Free Worlds and gives you <payment>.`
```
Comment (Amazinite) [inline data/boarding missions.txt]: As above, `The` shouldn't be capitalized.

File: data/boarding missions.txt [hunk 5]
```diff
@@ -37,3 +37,48 @@ mission "Assisting Hai"
...
+	assisting
+	repeat
+	to offer
+		random < 15
+	source
+		government "Militia"
+	on offer
+		payment 10000
+		conversation
+			`The captain of the <origin> is grateful for your service to The Free Worlds and gives you <payment>.`
```
Comment (Amazinite) [inline data/boarding missions.txt]: The militia and the Free Worlds aren't the same thing.

**Comment by Amazinite** [comment]
I'm looking for new assisting/boarding missions, but I'm not sold on these ones. Ref: #4139 for ideas that I threw out there.

**Pull request closed without merging**